templates/ocs-deployment/CephBlockPool.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: {{ cephblockpool_name | default('ocsci-cephblockpool') }}
+  namespace: {{ cluster_namespace | default('openshift-storage') }}
+spec:
+  failureDomain: {{ failureDomain | default('host') }}
+  replicated:
+    size: {{ size | default('3') }}
```

Comment:
it will work with just '3'?
I think it should be '3Gi'

Reply:
Ok my mistake.
So please change 'size' to 'replica_count'

Reply:
Forgot about this :) will be fixed in next commit